js/views/modals/Settings/Advanced/Settings.js

```diff
@@ -0,0 +1,108 @@
+import app from '../../../../app';
+import loadTemplate from '../../../../utils/loadTemplate';
+import baseVw from '../../../baseVw';
+import Appearance from './Appearance';
+import Transactions from './Transactions';
+import Server from './Server';
+import SMTPIntegration from './SMTPIntegration';
+import AppearanceSettings from '../../../../models/AppearanceSettings';
+import TransactionSettings from '../../../../models/TransactionSettings';
+import ServerSettings from '../../../../models/ServerSettings';
+import SMTPIntegrationSettings from '../../../../models/SMTPIntegrationSettings';
+
+export default class Advanced extends baseVw {
+  constructor(options = {}) {
+    super({
+      className: 'settingsAdvanced',
+      ...options
+    });
+
+    this.options = options;
+    this.settings = app.settings.clone( );
+
+
+    self.app = app;
+
+    this.listenTo(this.settings, 'sync', () => app.settings.set(this.settings.toModelFormatJSON()));
+    this.listenTo(this.settings, 'save', () => app.settings.set(this.settings.toModelFormatJSON()));
```

Comment:
Okay. I was feeling annoying when I was clicking save and not seeing the view update, even tho the settings were being sent to the server and saved there. Tho I think since it is a localhost API the lack of update must have been because something else wasn't being triggered correctly to render them. This 'save' event was more monkey patching attempt to get everything working together. I didn't actually believe there was a 'save' event, I was experimenting.